engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/bool/CompareToNode.java

```diff
@@ -0,0 +1,52 @@
+package org.enso.interpreter.node.expression.builtin.bool;
+
+import com.oracle.truffle.api.CompilerDirectives;
+import com.oracle.truffle.api.TruffleLanguage;
+import com.oracle.truffle.api.dsl.Cached;
+import com.oracle.truffle.api.dsl.CachedContext;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.Language;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.Context;
+import org.enso.interpreter.runtime.builtin.Ordering;
+import org.enso.interpreter.runtime.callable.atom.Atom;
+import org.enso.interpreter.runtime.error.PanicException;
+
+@BuiltinMethod(
+        type = "Boolean",
+        name = "compare_to",
+        description = "Comparison for Booleans.")
+public abstract class CompareToNode extends Node {
+    static CompareToNode build() { return CompareToNodeGen.create(); }
+
+    abstract Atom execute(Boolean _this, Object that);
+
+    @Specialization
+    Atom doBoolean(
+            Boolean _this,
+            Boolean that,
+            @CachedContext(Language.class) TruffleLanguage.ContextReference<Context> ctxRef,
```

Comment:
Can we avoid adding more deprecated calls. I think:
`ordering = Context.get(this).getBuiltins().ordering();`
should get the ordering you need in the new API.

Reply:
Sure, should I update all the other places where we use deprecated API, or leave it for a separate PR?